### DISCUSSION TITLE
ES6 support

### DISCUSSION THREAD

**Discussion by shruthikudlur** [dop251/goja#520] [General]
I see that the milestone for ES6 support is 100% complete. But the main page still says in progress. Can you please provide some clarity?
Thanks in advance

**Comment by dop251** [comment]
The milestone is there mostly for seeing what's already been done (by examining the closed issues). There is an open [issue](https://github.com/dop251/goja/issues/420) for documenting it better, but I don't currently have time to work on it. Any contributions will be welcome. For now, the best way to find out what's missing is to look at the skipped tests and blacklisted features in [tc39_test.go](https://github.com/dop251/goja/blob/master/tc39_test.go).
